app/views/portal/show.rb

```diff
@@ -783,154 +753,173 @@ def iiif_manifesto(identifier, collection)
       url_match || ids[identifier] || collections[collection]
     end
 
-    def media_items
-      aggregation = document.aggregations.first
-      return [] unless aggregation.respond_to?(:webResources)
+    def web_resource_media_item(web_resource)
```

Comment:
Assignment Branch Condition size for web_resource_media_item is too high. [32.95/15]<br>Method has too many lines. [37/10]